charts/opentelemetry-kube-stack/examples/cloud-demo/rendered/hooks.yaml

```diff
@@ -1,47 +1,5 @@
 ---
 # Source: opentelemetry-kube-stack/templates/hooks.yaml
-apiVersion: v1
```

Comment:
@TylerHelmuth idk why, but these got deleted from my local make run, despite a manual helm template not removing them.

Reply:
i think my helm version was outdated and mad at me! 🙇